### DISCUSSION TITLE
Limits for Github storage and Actions minutes

### DISCUSSION THREAD

**Discussion by corrados** [jamulussoftware/jamulus#922] [General]
We recently introduced to use Github Actions to check the code and create release installers. Github has limits for the storage and Actions minutes for free accounts, see: https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions. I.e., we have 500 MB and 2000 h/month available.

**Comment by pljones** [comment]
Is there any indication on how to display on-going statistics against these limits?

**Reply by ann0see** [reply]
Probably here: https://github.com/organizations/jamulussoftware/settings/billing

**Reply by pljones** [reply]
I get a 404 on that link...

**Reply by pljones** [reply]
> Organization owners and billing managers can view GitHub Actions usage for an organization.

That could be it.  I get to switch to `jamulussoftware` or my personal account but I can't manage `jamulussoftware`.

![image](https://user-images.githubusercontent.com/1549463/106370401-921ff200-6351-11eb-997d-2d8c6efcf914.png)
The "View organisation" button gets me nothing I can't see as my personal account.

**Reply by ann0see** [reply]
It should work now.

**Comment by npostavs** [comment]
I don't think Jamulus has to worry about it, the usage limits are only for private repositories:

https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration:

> GitHub Actions usage is free for both **public repositories** and self-hosted runners.

https://github.com/pricing:

> 2,000 Actions minutes/month 
> Free for public repositories
